firestore/tokens.py

```diff
@@ -0,0 +1,32 @@
+from fireo.models import Model
+from fireo.fields import IDField, TextField, DateTime
+from datetime import datetime, timedelta
+
+
+class Token(Model):
+    id = IDField(required=True)
+    token = TextField(required=True)
+    expiration_time = DateTime(required=True)
+
+
+def deleteToken(token_id):
+    if Token.collection.delete("token/" + token_id):
+        return True
+    return False
+
+
+def registerNewToken(id, token):
+    # return incident id
+    print("New Id:", id)
+    print("New Token:", token)
+    new_token = Token(
+        id=id,
+        token=token,
+        expiration_time=datetime.now() + timedelta(days=30),
```

Comment:
what does expiration time do?

Reply:
this means if user forgot to use the app for 30 days, we will not send notification anymore. This is not what we want. 

we want to stop sending notifications, only if notification sending failed. I think we can remove the expiration check.